src/components/Sidebar/NavContent.jsx

```diff
@@ -230,190 +150,63 @@ function NavContent({ handleDrawerToggle }) {
                       </Accordion>
                     </div>
                   </div>
-
-                  <Link
-                    component={NavLink}
-                    id="stake-nav"
-                    to="/"
-                    isActive={(match, location) => {
-                      return checkPage(match, location, "stake");
-                    }}
-                    className={`button-dapp-menu ${isActive ? "active" : ""}`}
-                    onClick={handleDrawerToggle}
-                  >
-                    <Typography variant="h6">
-                      <SvgIcon color="primary" component={StakeIcon} />
-                      <Trans>Stake</Trans>
-                    </Typography>
-                  </Link>
+                  <NavItem to="/stake" icon="stake" label={t`Stake`} onClick={handleDrawerToggle} />
 
                   {/* NOTE (appleseed-olyzaps): OlyZaps disabled until v2 contracts */}
-                  {/* <Link
-                    component={NavLink}
-                    id="zap-nav"
-                    to="/zap"
-                    isActive={(match, location) => {
-                      return checkPage(match, location, "zap");
-                    }}
-                    className={`button-dapp-menu ${isActive ? "active" : ""}`}
-                    onClick={handleDrawerToggle}
-                  >
-                    <Box display="flex" alignItems="center">
-                      <SvgIcon component={ZapIcon} color="primary" />
-                      <Typography variant="h6">OlyZaps</Typography>
-                    </Box>
-                  </Link> */}
+                  {/*<NavItem to="/zap" icon="zap" label={t`Zap`} /> */}
 
                   {EnvHelper.isGiveEnabled(location.search) ? (
-                    <>
-                      <Link
-                        component={NavLink}
-                        id="give-nav"
-                        to="/give"
-                        isActive={(match, location) => {
-                          return checkPage(match, location, "give");
-                        }}
-                        className={`button-dapp-menu ${isActive ? "active" : ""}`}
-                        onClick={handleDrawerToggle}
-                      >
-                        <Typography variant="h6">
-                          <SvgIcon color="primary" component={GiveIcon} />
-                          <Trans>Give</Trans>
-                          <SvgIcon component={NewIcon} viewBox="21 -2 20 20" style={{ width: "80px" }} />
-                        </Typography>
-                      </Link>
-                    </>
+                    <NavItem to="/give" icon="give" label={t`Give`} chip={t`New`} onClick={handleDrawerToggle} />
                   ) : (
                     <></>
                   )}
```

Comment:
not your code but why is this a thing?

Reply:
Good catch. For some reason that give feature flag was wrapped in an if/then, and not just a conditional if. I just swapped it to clean it up.